vcbuild.bat

```diff
@@ -327,21 +327,21 @@ if "%target%" == "Clean" goto exit
 @rem Skip signing unless the `sign` option was specified.
 if not defined sign goto licensertf
 
-call tools\sign.bat Release\node.exe
+call tools\sign.bat %node_exe%
 if errorlevel 1 echo Failed to sign exe&goto exit
 
 :licensertf
 @rem Skip license.rtf generation if not requested.
 if not defined licensertf goto stage_package
 
-%config%\node.exe tools\license2rtf.js < LICENSE > %config%\license.rtf
+%node_exe% tools\license2rtf.js < LICENSE > %config%\license.rtf
 if errorlevel 1 echo Failed to generate license.rtf&goto exit
 
 :stage_package
 if not defined stage_package goto install-doctools
 
 echo Creating package...
-cd Release
+cd out\Release
```

Comment:
The relative paths in the `copy`'s below and the `cd ..`'s (including the one in `:package_error` need to be adjusted. Maybe replace the `cd`'s with `pushd`/`popd`?

Reply:
Noticed that 😢 
Changed it into https://github.com/nodejs/node/blob/0e089b3f425036aef77fbd56c1de7ff7eba3d86f/vcbuild.bat#L326-L327